### PR TITLE
Dianomi Bid Adapter: add usersyncs

### PR DESCRIPTION
**Pull request by DianomiJH**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [x] Other

## Description of change

Update Dianomi's adapter to include Pixel usersyncs and update the maintainer email

- contact email of the adapter’s maintainer
prebid-maintainer@dianomi.com

### CODE REVIEW

File: modules/dianomiBidAdapter.js [hunk 1]
```diff
@@ -301,27 +301,30 @@ export const spec = {
...
+    if (syncOptions.iframeEnabled) {
+      // data is only assigned if params are available to pass to syncEndpoint
       return {
         type: 'iframe',
         url: `https://${endpoint}/prebid/usersync/index.html?${formatQS(params)}`,
       };
+    } else if (syncOptions.pixelEnabled) {
+      return {
+        type: 'image',
+        url: `https://${endpoint.includes('dev') ? 'dev-' : ''}data.dianomi.com/frontend/usync${formatQS(params)}`,
```
Comment (osazos) [inline modules/dianomiBidAdapter.js:326]: There is no `?` to define the start of the query string, is it expected?

**Review by osazos — COMMENTED** [review]
Hi @DianomiJH 

Please add tests for usersync support. 

Actually, the `data.dianomi.com/frontend/usync…` endpoint returns a 404 (and after local test, it seems missing the `?` in the url)

**Comment by osazos** [comment]
Hi @DianomiJH 

Thanks for your update.

There is still remaining the point about the endpoint `data.dianomi.com/frontend/usync…` which returns a 404. It should not.

**Comment by DianomiJH** [comment]
H @osazos , Just deployed a fix for this, this morning (I know, Friday deploys 😱 ) let me know if there's anything else

**Review by osazos — APPROVED** [review]
LGTM

**Pull request merged**